src/Common/test-runtime/LatestDependencies/project.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
@hongdai I would like to understand a little more about the build failure.  If I understand the intent of this PR, I believe it is very similar to what @dagood had working with #646. If this PR is attempting to solve the same issue then it makes sense to get dagood's solution working again, or build on it.  If they are solving different issues then ignore me.
Also, we regularly take corefx build script fixes, so if that solves any problems in this PR, we should probably do that first in a separate PR.  If you can tell me the issues, maybe I can take on that PR. Thoughts?